spec/enumerable_spec.rb

```diff
@@ -0,0 +1,92 @@
+require './main'
+
```

Comment:
### [OPTIONAL]
Using `require_relative` instead of `require` is usually a ruby convention. Kindly use require_relative when linking to files in your project.